packages/bento-design-system/src/Pagination/Config.ts

```diff
@@ -0,0 +1,11 @@
+import { ButtonProps, PaginationItemsPerPage } from "..";
+import { BentoSprinkles } from "../internal";
+
+export type PaginationConfig = {
+  paddingY: BentoSprinkles["paddingY"];
+  itemsPerPageOptions: Array<PaginationItemsPerPage>;
```

Comment:
> Also, the config merging will probably break this, as it will merge the arrays instead of replacing the default one with the new one. Please check if this is true.

It seems to me it correctly replaces the default array with the custom one, from a test in Storybook.

<img width="862" alt="Screenshot 2023-06-26 at 11 49 00" src="https://github.com/buildo/bento-design-system/assets/26444095/3e1071bb-9755-4897-8e74-8d3f02da7c26">

Reply:
Ok, I think the types are saving us here, but I suppose the actual config value contains the merged values. Could you try to `console.log` the config from the component?

Reply:
Uhm I forgot to overwrite the config, so the config values were the default ones (`10, 25, 50, 100`), but the type was the custom one (`5, 10, 25, 50`), without any error.
If I overwrite the config, the values are indeed merged, like you said.